.github/workflows/release-tag.yaml

```diff
@@ -0,0 +1,17 @@
+name: Push Tag to NPM
```

Comment:
Let's improve these names. Also the file name.

Reply:
Updated